tests/test_security_curve.cpp

```diff
@@ -208,8 +211,13 @@ int main (void)
     assert (rc == 0);
 
 #ifdef ZMQ_BUILD_DRAFT_API
-    int event = get_monitor_event (server_mon, NULL, NULL);
+    int event = get_monitor_event (server_mon, NULL, NULL, 0);
     assert (event == ZMQ_EVENT_HANDSHAKE_SUCCEED);
+
+    // This event has to be the last one
+    Sleep(250);
```

Comment:
I committed this `Sleep` by mistake, it will be removed in the next one, but I tested locally the behavior I describe in my comment bellow is the same with and without this `Sleep`